etna/whatson/models.py

```diff
@@ -0,0 +1,633 @@
+    ]
+
+    content_panels = BasePageWithIntro.content_panels
+
+
+class EventPage(ArticleTagMixin, TopicalPageMixin, BasePageWithIntro):
+    """EventPage
+
+    A page for an event.
+    """
+
+    # Content
+    lead_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    # Event information
+    event_type = models.ForeignKey(
+        EventType,
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    # Start and end date will be brought in from the API when we have it.
+    start_date = models.DateTimeField(
+        verbose_name=_("start date"),
+        null=True,
+        blank=True,
+        help_text=_("The date and time the event starts."),
+    )
+
+    end_date = models.DateTimeField(
+        verbose_name=_("end date"),
+        null=True,
+        blank=True,
+        help_text=_("The date and time the event ends."),
+    )
+
+    description = RichTextField(
+        verbose_name=_("description"),
+        null=True,
+        blank=True,
+        help_text=_("A description of the event."),
+    )
+
+    useful_info = RichTextField(
+        verbose_name=_("need to know"),
+        null=True,
+        blank=True,
+        help_text=_("Useful information about the event."),
+    )
+
+    target_audience = RichTextField(
+        verbose_name=_("who it's for"),
+        null=True,
+        blank=True,
+        help_text=_("Info about the target audience for the event."),
+    )
+
+    # Venue information
+    venue_type = models.CharField(
+        max_length=15,
+        verbose_name=_("venue type"),
+        null=False,
+        blank=False,
+        choices=VenueType.choices,
+        default=VenueType.IN_PERSON,
+        help_text=_("The type of venue for the event."),
+    )
+
+    venue_website = models.URLField(
+        max_length=255,
+        verbose_name=_("venue website"),
+        null=True,
+        blank=True,
+        help_text=_("The website for the venue."),
+    )
+
+    venue_address = RichTextField(
+        verbose_name=_("venue address"),
+        null=True,
+        blank=True,
+        help_text=_("The address of the venue."),
+    )
+
+    venue_space_name = models.CharField(
+        max_length=255,
+        verbose_name=_("venue space name"),
+        null=True,
+        blank=True,
+        help_text=_("The name of the venue space."),
+    )
+
+    video_conference_info = RichTextField(
+        verbose_name=_("video conference info"),
+        null=True,
+        blank=True,
+        help_text=_("Information about the video conference."),
+    )
+
+    # Booking information
+    booking_type = models.CharField(
+        max_length=20,
+        verbose_name=_("booking type"),
+        null=False,
+        blank=False,
+        default="Drop in",
+        help_text=_("The type of booking for the event."),
+    )
```

Comment:
I have some worries about this one. Definitely feels like a `TextChoices` would be handy here for consistency

Reply:
If unclear, you could just remove for now and add later, once better defined

Reply:
I think on this one, as we're populating it via the API - I wasn't going to make it a TextChoices or anything (or have it editable)

I was going to populate it via enums from the API, and make it a non-editable field. Do you think we should make this editor-editable?